packages/core/src/components/Checkbox/__tests__/checkbox-tests.jest.tsx

```diff
@@ -122,7 +134,8 @@ describe("Checkbox tests", () => {
         option1Value,
         checkbox1Name,
         checkbox2Name,
-        checkbox3Name
+        checkbox3Name,
+        defaultChecked: true
```

Comment:
What is this?

Reply:
In the existing tests, the tests had defaultChecked set to true, I needed it to be false in my tests so I can check the autoFocus so I added this as optional